.gitignore

```diff
@@ -44,4 +44,9 @@ next-env.d.ts
 # storybook
 build-storybook.log
 
-**/dist/**
+# storybook generated doc
+/packages/tokens/docs/
+/public/design-tokens.source.json
```

Comment:
This json file is generated by `storybook-design-token`?

Reply:
Yes, and I don't know why or how to make it stop rendering this file